pkgs/tools/misc/lice/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, python3Packages }:
+
+python3Packages.buildPythonPackage rec {
+
+  package = "lice";
+  version = "0.4";
+  name = "${package}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "licenses";
+    repo = "lice";
+    rev = "71635c2544d5edf9e93af4141467763916a86624";
```

Comment:
Please use the tag corresponding to the release. In this case `rev = version` should work.